src/core/object/controller/controller.ts

```diff
@@ -97,8 +97,8 @@ export default class Controller {
 		}
 		return (
 			this.currentSong?.parsed.isScrobblingAllowed &&
-			(await this.blocklist.shouldScrobbleChannel(
-				this.connector.getChannelId?.(),
+			!(await this.blocklist.getChannelLabel(
```

Comment:
I feel the sematics of this is a lot harder to read now. Like the channel label dictacts if scrobbling is allowed? Can we add a util function that is called `shouldScrobbleChannel` back again?

Reply:
Done